rpc/src/util.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+/// Helper macros. Used to simplify adding new RpcHandler to Registry
+/// `registry` - name of local registry variable
+/// `method` - name of new rpc method
+/// `num_args` - number of method arguments
+macro_rules! register_rpc_method {
```

Comment:
is a macro necessary here? why not a regular function? not sure what the advantage of using a macro over a function is here

Reply:
so macro is used here to provide easier registry registration for devs who implement new rpc methods
main pain point is around converting convenient `async fn ...` to suitable representation (see RpcHandler type definition). Let me know if you find better approach though